helm/ph-ee-engine/connector-bulk/templates/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
 {{ toYaml .Values.deployment.annotations | indent 8 }}
 {{- end }}
     spec:
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
```

Comment:
Are we updating values.yaml for these values?
If no, what is the impact of leaving these values blank

Reply:
In Values.yaml it was already present but the configuration was not present in the deployment file of the service.

Reply:
https://github.com/openMF/ph-ee-env-template/blob/3cfff59f6784ab8f41309db4215dabbc53cad43f/helm/ph-ee-engine/values.yaml#L1580